diff --git a/build/cmake/pkg_config.cmake b/build/cmake/pkg_config.cmake
index e8fff2e77..e71e13b20 100644
--- a/build/cmake/pkg_config.cmake
+++ b/build/cmake/pkg_config.cmake
@@ -62,7 +62,7 @@ file(APPEND "${pkgconfig_file}" "\nConflicts:\n")
 file(APPEND "${pkgconfig_file}" "Libs: -L\${libdir} -l${pkg_name}\n")
 if(CONFIG_MULTITHREAD AND HAVE_PTHREAD_H)
   file(APPEND "${pkgconfig_file}" "Libs.private: -lm -lpthread\n")
-else()
+elseif(not WIN32)
   file(APPEND "${pkgconfig_file}" "Libs.private: -lm\n")
 endif()
 file(APPEND "${pkgconfig_file}" "Cflags: -I\${includedir}\n")
